.github/ISSUE_TEMPLATE/questions_help.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Questions/Help
-about: Submit questions or requests for help regarding downgrade
-labels: 'question, needs triage'
+about: Submit questions or requests for help
+labels: 'question, triage'
```

Comment:
That's weird that it's a comma-separated String and not an actual Yaml Array.

Reply:
Hmm I should have probably made it differently, did not realize there was so much customization possible:

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

Anw will get to that another time